scala/scala.bzl

```diff
@@ -384,3 +424,190 @@ def scala_repositories():
     url = "https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.11/2.2.6/scalatest_2.11-2.2.6.jar",
     sha256 = "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
+
+
+# TODO is it possible to share scala-library with the above?
+def scrooge():
```

Comment:
I mean, I can. I guess I can create a fake source jar and test off of that. The nice thing about having the two together right now is that it forces me to deal with the compile dependencies properly, etc. So I'm happy doing this, but I'm afraid that decoupling it will make it harder to tackle the issues around properly passing through the compile time and run time dependencies.

What do you think?